src/main/java/com/denizenscript/clientizen/access/KeyBindingMixinAccess.java

```diff
@@ -0,0 +1,8 @@
+package com.denizenscript.clientizen.access;
+
+public interface KeyBindingMixinAccess {
+
+    void disableUntilPress();
+
+    void forceSetPressed(boolean pressed);
```

Comment:
From what I've seen, it's best practice to prefix these with `clientizen$` too. Not sure if it has any effect though